docker-compose/docker-compose.yaml

```diff
@@ -31,8 +31,6 @@ services:
       - 50000:8080
 
   amenities:
-    build: ../src/amenities
-    ports:
-      - 50004:8080
-    environment:
-      - forProduction=test
\ No newline at end of file
+    extends:
+      file: ../src/amenities/amenities_server/docker-compose.yaml
```

Comment:
I would prefer the `docker-compose.yaml` to be located in the same folder as the `Dockerfile`, within the `/amenities` directory. This keeps all related configuration files together, making it easier to manage and maintain.
Just a suggestion. But if you do decide to change, please change the README as well